Client/Frontend/Library/Bookmarks/BookmarksPanel.swift

```diff
@@ -20,6 +21,7 @@ class BookmarksPanel: SiteTableViewController, LibraryPanel, CanRemoveQuickActio
     struct UX {
         static let FolderIconSize = CGSize(width: 24, height: 24)
         static let RowFlashDelay: TimeInterval = 0.4
+        static let BookmarksHelpScoutUrlString = "https://ecosia.helpscoutdocs.com/article/458-import-export-bookmarks"
```

Comment:
Usually we have those kind of URLs in [Core/Environment.swift](https://github.com/ecosia/ios-core/blob/main/Sources/Environment/Environment.swift) to have a central overview of the used URLs. Would you mind moving it there? 🙏

Reply:
Oh, that sounds reasonable, I'll move it there then.